package.json

```diff
@@ -13,8 +13,11 @@
     "@emotion/react": "^11.13.3",
     "@emotion/styled": "^11.13.0",
     "@mui/icons-material": "^6.1.3",
+    "@mui/lab": "^6.0.0-beta.14",
```

Comment:
please remove Mui labs as it is not in use

Reply:
done